app/views/how-tos/switching-from-govuk-prototype-kit.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Switching from the GOV.UK prototype kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
+        Switching from the GOV.UK prototype kit
+      </h1>
+
+
+<p>
+      If you have learned how to use the GOV.UK prototype kit, you will be able to use the NHS.UK prototype kit. This is because the two prototype kits are built in the same way. However, you must do some things differently.
+    </p>
+<h2>Installing and running the NHS.UK prototype kit</h2>
+
+<p>
+When you are installing and running your prototype:
+
+{{ list({
+  title: "Do",
+  type: "tick",
+  items: [
+    {
+      item: 'check when you install your prototype that you are using the right version of node.js - you may need to install and use a different version. If you need to make prototypes that for both NHS.UK and GOV.UK you can switch versions of node with a <a href="https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/">node version manager</a>'
+    },
+    {
+      item: 'start your local server by entering in the terminal: <pre class="app-pre"><code class="language-markup">npm run watch</code></pre> '
+    }
+  ]
+}) }}
+
+{{ list({
+  title: "Don't",
+  type: "cross",
+  items: [
+    {
+      item: 'start your local server the same way as in the GOV.UK prototype kit as it will not work'
+    }
+  ]
+}) }}
+
+
+
+<h2>Making pages</h2>
+
+<p>
+  When you are making pages:
+</p>  
+
+{{ list({
+  title: "Do",
+  type: "tick",
+  items: [
+    {
+      item: 'use the NHS.UK design system for <a href="https://service-manual.nhs.uk/design-system/styles">styles</a>, <a href="https://service-manual.nhs.uk/design-system/components">components</a> and <a href="https://service-manual.nhs.uk/design-system/patterns">patterns</a>'
+    },
+    {
+      item: 'use <a href="https://service-manual.nhs.uk/design-system">NHS.UK macro naming conventions</a>, for example <pre class="app-pre"><code class="language-markup">radios</code></pre>'
+    },
+    {
+      item: 'use <a href="https://service-manual.nhs.uk/design-system/styles/typography">NHS.UK typography classes</a>, for example <pre class="app-pre"><code class="language-markup">' + '<p class="nhsuk-body">nhsuk-body</p>' | escape  + '</code></pre>'
+    },
+    {
+      item: ' be aware that the NHS.UK prototype kit may not have functionality that is in the GOV.UK one, for example, plugins      '
```

Comment:
Some plugins do still kinda work, eg if they’re defining filters, so perhaps worth noting this? They have to be referenced the older way though.

```suggestion
      item: ' be aware that the NHS.UK prototype kit does not have some functionality that is in the GOV.UK one, for example the interface to create pages from templates'
   },
   {
      item: ' note that plugins designed for the GOV.UK Prototype Kit may not work or may need to be installed differently. For example, the [GOV.UK Prototype Kit Filters](https://x-govuk.github.io/govuk-prototype-filters/get-started/) plugin can be installed using the instructions for earlier versions of the kit, which requires making some changes to `filters.js`.'
```